oid4vc/oid4vc/routes.py

```diff
@@ -1221,6 +1438,10 @@ async def register(app: web.Application):
             web.get("/oid4vp/presentations", list_oid4vp_presentations),
             web.get("/oid4vp/presentation/{request_id}", get_oid4vp_pres_by_id),
             web.delete("/oid4vp/presentation/{presentation_id}", oid4vp_pres_remove),
+            web.post("/oid4vp/dcql/query-create", create_dcql_query),
```

Comment:
Nitpick: the verb POST already kind of implies a creation. So I'd recommend using `/oid4vp/dcql/queries` as the path. This would line up better with the presentation definition endpoints, too.